docs/pages/versions/unversioned/sdk/screen-capture.mdx

```diff
@@ -17,6 +17,13 @@ import { SnackInline } from '~/ui/components/Snippet';
 
 This is especially important on Android since the [`android.media.projection`](https://developer.android.com/about/versions/android-5.0.html#ScreenCapture) API allows third-party apps to perform screen capture or screen sharing (even if the app is in the background).
 
+On Android, the screen capture callback works without additional permissions only for Android 14+. **You don't need to request or check permissions for blocking screen capture or using the callback on Android 14+.**
+
+If you want to use the screen capture callback on Android 13 or lower, you need to add the `READ_MEDIA_IMAGES` permission to your `AndroidManifest.xml` file.
+You can do that using the `android.permissions` key in your app config. [Learn more.](https://docs.expo.dev/guides/permissions/#android)
```

Comment:
```suggestion
If you want to use the screen capture callback on Android 13 or lower, you need to add the `READ_MEDIA_IMAGES` permission to your `AndroidManifest.xml` file. You can use the `android.permissions` key in your app config. See [Android permissions](/guides/permissions/#android) for more information.
```

Let's use description for the link instead of learn more as per writing style guide.